gallery/wizardlm2-8x22b-q4_k_s.yaml

```diff
@@ -0,0 +1,37 @@
+backend: llama-cpp
```

Comment:
by adding the file into `gallery/` won't make it display in the index. You need to add the model into the index first.

See an example here: https://github.com/mudler/LocalAI/blob/c50e0edcb80c92d3048263b16b05deaa05148d33/gallery/index.yaml#L3064

You can then refer from the entries to the configuration file inside the gallery folder. However, I'd keep the file content minimal so can be override by the entries in the index.